### ISSUE TITLE
can it be used for tabular data ?

### ISSUE THREAD

**Issue by Sandy4321** [FloList/EMPL#1]
can it be used for tabular data ?

like
https://towardsdatascience.com/deep-quantile-regression-in-tensorflow-1dbc792fe597

**Comment by FloList** [comment]
Hi Sandy,
This Github repo is specifically for the task of _histogram_-valued regression (so the label belonging to each input should be an entire histogram), and a neural network trained with the Earth Mover's Pinball Loss will estimate the quantiles of the cumulative histogram belonging to each input. If you're looking for a loss function to do quantile regression for _scalar_ labels as in the blog post you mentioned, you can directly take the pinball loss (or quantile loss), for example here:
[https://github.com/strongio/quantile-regression-tensorflow/blob/master/Quantile%20Loss.ipynb](https://github.com/strongio/quantile-regression-tensorflow/blob/master/Quantile%20Loss.ipynb).
Also, the pinball loss function is available as a Tensorflow [Add-on](https://www.tensorflow.org/addons/api_docs/python/tfa/losses/PinballLoss).
Cheers,
Florian

**Comment by Sandy4321** [comment]
thanks for answer
for 
Also, the pinball loss function is available as a Tensorflow [Add-on](https://www.tensorflow.org/addons/api_docs/python/tfa/losses/PinballLoss).
is written 

Warning: This project is deprecated. TensorFlow Addons has stopped development, The project will only be providing minimal maintenance releases until May 2024. See the full announcement [here](https://www.tensorflow.org/addons) or on [github](https://github.com/tensorflow/addons/issues/2807).

**Comment by Sandy4321** [comment]
it is questionabe if
https://github.com/strongio/quantile-regression-tensorflow/blob/master/Quantile%20Loss.ipynb

will work for new TF versions
